extension.js

```diff
@@ -388,6 +410,9 @@ function activate(context) {
     storage.set("patchedBefore", false);
   });
 
+  disposable = context.subscriptions.push(vscode.commands.registerCommand('tabscolor.clearAllColors', () => {
```

Comment:
same here, rename to clearOpenTabsColors

Reply:
Function name updated